src/otx/core/data/transform_libs/mmcv.py

```diff
@@ -0,0 +1,76 @@
+"""Helper to support MMCV data transform functions."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Callable
+
+import numpy as np
+from mmcv.transforms import LoadImageFromFile as MMCVLoadImageFromFile
+from mmcv.transforms.builder import TRANSFORMS
+
+from otx.core.data.entity.base import OTXDataEntity
+from otx.core.utils.config import convert_conf_to_mmconfig_dict
+
+if TYPE_CHECKING:
+    from mmengine.registry import Registry
+
+    from otx.core.config.data import SubsetConfig
+
+
+@TRANSFORMS.register_module(force=True)
+class LoadImageFromFile(MMCVLoadImageFromFile):
+    """Class to override MMCV LoadImageFromFile."""
+
+    def transform(self, entity: OTXDataEntity) -> dict | None:
+        """Transform OTXDataEntity to MMCV data entity format."""
+        img = entity.image
+
+        if self.to_float32:
+            img = img.astype(np.float32)
+
+        results = {}
+        results["img"] = img
+        results["img_shape"] = img.shape[:2]
+        results["ori_shape"] = img.shape[:2]
+
+        results["__otx__"] = entity
+
+        return results
+
+
+class MMCVTransformLib:
+    """Helper to support MMCV transforms in OTX."""
+
+    @classmethod
+    def get_builder(cls) -> Registry:
```

Comment:
What is the purpose of `get_builder` method that simply returns TRANSFORMS? Could users not directly call `TRANSFORMS.build(cfg)` within the generate method? This class method doesn't seem to add significant value to the class.

Reply:
It has a significant value to reuse the base `generate()` with `mmdet`'s `TRANSFORMS`.
https://github.com/openvinotoolkit/training_extensions/blob/8c8c50f64571c4b25ddcf5a1276ebaba5a4bcca9/src/otx/core/data/transform_libs/mmdet.py#L97